ksql-parser/src/main/java/io/confluent/ksql/parser/rewrite/StatementRewriter.java

```diff
@@ -397,16 +397,7 @@ protected Node visitCreateStream(final CreateStream node, final Object context)
         .map(tableElement -> (TableElement) process(tableElement, context))
         .collect(Collectors.toList());
 
-    final Map<String, Literal> rewrittenProps = node
-        .getProperties()
-        .entrySet()
-        .stream()
-        .collect(Collectors.toMap(
-            Entry::getKey,
-            e -> (Literal) process(e.getValue(), context)
-        ));
-
-    return node.copyWith(rewrittenElements, rewrittenProps);
+    return node.copyWith(rewrittenElements, node.getProperties());
```

Comment:
This is no longer allowing the literals to be re-written. Is this intentional?

Reply:
yes it is - I noticed that nowhere was using this and I don't think it really makes sense to arbitrarily rewrite literals with a rewriter. If you want to rewrite the properties, I would suggest doing it manually. If you feel strongly about this I can rework it but it's not a simple task.